File: Completed/BuffaloBuffalo/BuffaloBuffalo.py
class BuffaloBuffalo:
    def check(self, s):
        lst=list(set(s.split()))
        if len(lst) != 1:
            return "Not good"
        elif lst[0] != "buffalo":
            return "Not good"
        elif s.count(' ') != len(s.split())-1:
            return "Not good"

File: Completed/BuffaloBuffalo/BuffaloBuffalo.cpp
class BuffaloBuffalo {
    public:
    string check(string s) {
        int n=s.length();
        if(n%8!=7) { return "Not good"; }
        
        string b="buffalo ";
        
        for(int i=0; i<n; i++) {
            if(b[i%8]!=s[i]) { return "Not good"; }
        }
        return "Good";
    }
};
